source/base/ReferenceCounter.ooc

```diff
@@ -22,6 +21,12 @@ ReferenceCounter: class {
 		this _lock = Mutex new(mutexType)
 //		"  RC, #{This count}, #{this _target class name}" println()
 	}
+	free: override func {
+		This count -= 1
+//		"--RC, #{This count}, #{this _target class name}" println()
```

Comment:
I was actively avoiding that and other style fixes, since I did not want to clutter the commits. Should I clean up as much as I can, then?